win32ss/user/ntuser/nonclient.c

```diff
@@ -394,11 +394,13 @@ DefWndDoSizeMove(PWND pwnd, WORD wParam)
       if (msg.message == WM_KEYDOWN && (msg.wParam == VK_RETURN || msg.wParam == VK_ESCAPE))
          break; // Exit on Return or Esc
 
-      if (!g_bWindowSnapEnabled && (msg.message == WM_LBUTTONUP))
+      if (!g_bWindowSnapEnabled && (msg.message == WM_LBUTTONUP ||
+         (msg.message == WM_MOUSEMOVE && (msg.wParam & MK_LBUTTON) != MK_LBUTTON)))
```

Comment:
```suggestion
      if (!g_bWindowSnapEnabled &&
          (msg.message == WM_LBUTTONUP ||
           (msg.message == WM_MOUSEMOVE && (msg.wParam & MK_LBUTTON) != MK_LBUTTON)))
```

Reply:
In the original layout the if test line break is on the only "||" which makes better sense to me. Thanks.